src/renderer/views/preload/PreloadProgressView.tsx

```diff
@@ -29,7 +28,7 @@ const PreloadProgressView = observer(() => {
   } = useNodeExceptionSubscription();
   const preloadProgress = preloadProgressSubscriptionResult?.preloadProgress;
 
-  const [isPreloadEnded, setPreloadStats] = useState(false);
+  const [isPreloadEnded, setIsPreloadEnded] = useState(false);
```

Comment:
It isn't a big deal, but IMHO `preloadEnded` and `setPreloadEnded` are more natural for me.